[1,"Error occurred during cleanup: Error: The process cannot access the file because it is being used by another process. (os error 32): remove 'C:\\Users\\azad\\AppData\\Local\\Temp\\quarto-session3fdec7f548aaf570'\n"]
